core/tests/protractor/editorAndPlayer.js

```diff
@@ -18,15 +18,17 @@
 
 var general = require('../protractor_utils/general.js');
 var forms = require('../protractor_utils/forms.js');
-var users = require('../protractor_utils/users.js');
+var UsersPage = require('../protractor_utils/UsersPage.js');
 var workflow = require('../protractor_utils/workflow.js');
 var editor = require('../protractor_utils/editor.js');
 var player = require('../protractor_utils/player.js');
 
+var usersPage = new UsersPage.UsersPage()
```

Comment:
Please move this into a `beforeAll` call in the `describe` block. Also add the semi-colon missing.

Reply:
done